spec/extensions/presence_spec.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
-require "rspec_helper"
-require_relative "../initialize/presence"
+require_relative "../rspec_helper"
```

Comment:
This spec file is testing the presence monkeypatches, which were removed.  I think this spec can be removed, too.

Reply:
Yeah, I leave it there only for the reviewer but now that's approved I'll submit the removal